munit/shared/src/main/scala/munit/Assertions.scala

```diff
@@ -37,7 +38,7 @@ trait Assertions {
   )(implicit loc: Location): Unit = {
     StackTraces.dropInside {
       if (!cond) {
-        throw new DottyBugAssumptionViolatedException(munitPrint(clue))
```

Comment:
Should we delete `DottyBugAssumptionViolatedException`?

Reply:
Nevermind, the diff was collapsed. I see it's deleted now

Reply:
didn't I do it?